integration-tests/contracts/contract_deployer.go

```diff
@@ -763,6 +764,25 @@ func (e *EthereumContractDeployer) DeployUpkeepTranscoder() (UpkeepTranscoder, e
 	}, err
 }
 
+func (e *EthereumContractDeployer) LoadUpkeepTranscoder(address common.Address) (UpkeepTranscoder, error) {
```

Comment:
there are multiple versions of transcoder but maybe this should deploy the most recent version 4.0?

Reply:
Good point. Created task - https://smartcontract-it.atlassian.net/browse/AUTO-7455

Reply:
just a note: 4.0 only works with upkeeps for 2.1 registry. 3.0 works with 2.0 registry. but i don't see a need to migrate to 2.0.